index.html

```diff
@@ -55,8 +180,93 @@ <h1>I'm Nohain. <br />Glad to see you!</h1>
               <a href="#"><i class="fa-brands fa-angellist"></i></a>
             </li>
           </ul>
-        </section>
+          <input
+            class="btn"
+            id="resume-btn"
+            type="button"
+            value="Get my resume"
+          />
+        </div>
+
+        <div class="bottom-2">
```

Comment:
- [ ] I kindly suggest that you make use a two level nested list here i.e make this `div` a `ul` and create and `li` tag container for each `skill-type`

## EXAMPLE
```
   <ul>
        <li class="languague">
              <ul>
                  <li><a href="#">JavaScript</a></li>
                  <li><a href="#">HTML</a>
                  <li><a href="#">CSS</a>
             </ul>
         </li>
          <li class="framework">
              <ul>
                  <li><a href="#">Bootstrap</a></li>
                  <li><a href="#">Ruby on Rails</a>
                  <li><a href="#">RSPEC</a>
             </ul>
         </li>
            <li class="skills">
              <ul>
                  <li><a href="#">Code Kit</a></li>
                  <li><a href="#">GitHub</a>
                  <li><a href="#">Codepen</a>
             </ul>
         </li>                                     
</ul>
```

Please note that the above is just an example and may not apply directly to your code.

Reply:
Complete.